core/unit_test/incremental/Test12b_TeamScratch.hpp

```diff
@@ -68,7 +68,7 @@ struct TeamScratch {
 
     Kokkos::parallel_for(
         "Team",
-        policy_t(pN, Kokkos::AUTO)
+        policy_t(pN, Kokkos::AUTO, 1)
```

Comment:
Did you mean to change the vector length?

Reply:
See above. 1 is the default.